controllers/admin/AdminModulesController.php

```diff
@@ -124,10 +124,10 @@ public function __construct()
     public function checkCategoriesNames($a, $b)
     {
         if ($a['name'] === $this->trans('Other Modules')) {
-            return true;
+            return 0;
         }
 
-        return (bool) ($a['name'] > $b['name']);
+        return strlen($a['name']) <=> strlen($b['name']);
```

Comment:
I think there is a misunderstanding here.
If you see the following snippet you'll see that the old logic is not the same as the new:
```php

(bool) ('d' > 'abcd')	//=> true

strlen('d') <=> strlen('abcd')	//=> -1

```
You just need to remove `strlen` as before it was not the lengths which was compared but the `>` comparator on string proceed as an alphabetic order comparison. So I think the following code is more appropriate
```php
'd' <=> 'abcd'	//=> 1
```

Reply:
makes sense :) I'll change that